marshmallow/utils.py

```diff
@@ -52,6 +52,11 @@ def is_generator(obj):
     return inspect.isgeneratorfunction(obj) or inspect.isgenerator(obj)
 
 
+def is_iterable(obj):
```

Comment:
See previous comment: not sure this is necessary--I think you can just use `is_iterable_but_not_string`.

Reply:
Will remove if it's not needed.